examples/app/ohlc/main.py

```diff
@@ -95,5 +93,7 @@ def update(t):
 
     source.stream(new_data, 300)
 
+curdoc().add_root(column(row(mean, stddev, mavg), gridplot([[p], [p2]], toolbar_location="left", plot_width=1000)))
+
```

Comment:
I think it's better to demonstrate `curdoc().add_root` as late as possible, as a best practice.  The resolution of model reference happens at `add_root` not at serialization, so confusing situations can happen if `add_root` happens before a model has changes that affect its references.